### ISSUE TITLE
Associate Kathleen Kay's profile with University of California, Santa Cruz

### ISSUE THREAD

**Issue by jleighherzog** [CDL-Dryad/dryad-product-roadmap#1388]
Ryan: I already put in different payment for this, but I think it needs to be waived in Stripe. (see comments under the full ticket)

Please associate user's profile with University of California, Santa Cruz: 

Current email: kmkay@ucsc.edu 
ORCID ID: 0000-0001-8858-110X
Dataset for reference: 10.5061/dryad.tb2rbp00h

**Comment by sfisher** [comment]
I changed her user account to be at UCSC.

I updated her payment in our database to be covered by UCSC.  (her former stripe payment id is in_1JDhhkKX6qdZziO2Beb1MFQs).  I think Ryan may need to go in to stripe and waive payment or do something manually when he's back.

However we cannot update this dataset to be under UCSC because they use EZID and different DOI ranges rather than Datacite and the DOI format which Dryad uses.  The UCs are a little too special with their DOIs and the choices aren't good for situations like this.  :-/ 

1. We could change her DOI to be one that would work under UCSC but that would break all her links (bad) and DOIs are supposed to be like diamonds (forever).
2. We could just change that dataset to be under UCSC anyway, but then anytime it tried to update it would fail all the updates since UCSC handles DOIs differently and it would try to update the special UCSC way.

So I think the best solution is to leave the dataset under Dryad and waive the payment manually.

For other partners we can do some moving around since they don't use different DOI services, but not the UCs because of their very special DOIs.  So for people at the UCs we can hope they authenticate their account when they create it 🤞  so they don't want to move things around later.
